client/server/pages/test/analytics.js

```diff
@@ -10,7 +10,7 @@ import sinon from 'sinon';
  * Internal dependencies
  */
 import analytics from '../../lib/analytics';
-import { logSectionResponse } from 'pages/analytics';
+import { logSectionResponse } from 'calypso/server/pages/analytics';
```

Comment:
It was working before because it looks for packages in `client/server/` (https://github.com/Automattic/wp-calypso/blob/trunk/test/server/jest.config.js#L3)